index.html

```diff
@@ -137,66 +173,101 @@ <h2 class="cont-header">Professional Art Printing Data</h2>
             </div>
-            <div class="skillscont">
-                <div class="lang-heading"><img src="images/circle.png" alt="">
-                    <h3 class="iconic"></h3>Skills</div>
-                <ul class="langlist">
-                    <li class ="jss">Codekit</li>
-                    <li class="jss">Github</li>
-                    <li class="jss">Codepen</li>
-                    <li class="jss"> Gitlab</li>
-                    <li class="jss"> Terminal</li>
-                </ul>
+
+            <div class="underline"></div>
+            <div class="lang-cont">
+                <div class="language-container">
+                    <div class="lang-heading"><img src="images/rhombus.png" alt="">
+                        <h3 class="iconic">Language</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Javascript</li>
+                        <li class="jss">Ruby</li>
+                        <li class="jss">Html</li>
+                        <li class="jss"> Css</li>
+                    </ul>
+                </div>
+                <div class="framecont">
+                    <div class="lang-heading"><img src="images/rectangle.png" alt="">
+                        <h3 class="iconic">Framework</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Bootstrap</li>
+                        <li class="jss">Ruby on Rails</li>
+                        <li class="jss">RSPec</li>
+                        <li class="jss">CapyBara</li>
+                        <li class="jss">Selenium</li>
+                    </ul>
+                </div>
+                <div class="skillscont">
+                    <div class="lang-heading"><img src="images/circle.png" alt="">
+                        <h3 class="iconic"></h3>Skills
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Codekit</li>
+                        <li class="jss">Github</li>
+                        <li class="jss">Codepen</li>
+                        <li class="jss"> Gitlab</li>
+                        <li class="jss"> Terminal</li>
+                    </ul>
+                </div>
             </div>
-        </div>
 
     </section>
 
+    <form action="https://formspree.io/f/mlevdeoa" method="post" class="contact-form">
+        <div class="interest">
+            <h5>I'm always interested in hearing about new projects,
+                so if you'd like to chat please get in touch.</h5>
+        </div>
+        <div class="all">
+            <div><input type="text" placeholder="Full name" maxlength="30" class="inputs" name="user-name" required>
+            </div>
+            <div><input type="email" placeholder="Email" class="inputs" name="e-mail" required></div>
+            <div class="text-cont"><textarea name="message" placeholder="Enter your text here" maxlength="500"
```

Comment:
- [ ] Great improvement 👏. However, as suggested by the previous reviewer, please try to match your form input styles with the styles shown in the Figma design. To be specific, please try to remove the borders on the `<textarea>`  so that it matches the styles in the Figma design 😃 . Please look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/186245355-b48d2654-b52d-4c38-84ab-1945fdb4eee3.png)

**Figma design**
![image](https://user-images.githubusercontent.com/90222110/186245625-3b30ffc7-e22b-4a64-952f-5e0a2fa5280f.png)